### ISSUE TITLE
Multiple pathologies

### ISSUE THREAD

**Issue by Luugaaa** [ivadomed/model_seg_sc-gm-lesion_human_ms_exvivo_t2star#34]
# Context

Haris reviewed the paper, we know have new critical information.
The dataset is actually comprised of cases with 4 possible pathologies: multiple sclerosis (MS; 18 cases), progressive multifocal leukoencephalopathy (PML; 4 cases), amyotrophic lateral sclerosis (ALS; 2 cases), and cerebrovascular disease (2 cases). We didn't work with ALS and cerebrovascular disease at all (by chance). 

# What we did (before knowing this)
The cases we worked with have MS (8/18 cases) and PML (all 4/4 cases); again, by chance.
For each experiment we trained 4 folds and averaged the results of those 4 folds on their validation sets. Out of the 4 folds, 2 folds have PML cases in their validation set (that's a good news, and it sounds ok since with run inference using ensembling). 

# Problem
There is no PML left to test on. Re-running all experiments while putting 1 PML case aside is not a good approach (would take weeks). 
Since we tested on MS data, we should probably have the narrative: "We trained and optimized a model for MS segmentation, we used PML during training to increase the generalization power of our model (similar lesions)." Problem: we have no experiment that supports this + it might not even be true.

**Comment by Luugaaa** [comment]
@jcohenadad I think we should have a one-on-one to talk about this issue
